### DISCUSSION TITLE
Clan Boss Tool

### DISCUSSION THREAD

**Discussion by Escondido** [FarbstoffRSL/RSL-Helper#425] [General]
Does anybody know how to turn off a skill when using the CB tool?  For example,  Aniri only uses her A3 if somebody die, rhazin doesn't use his A3, etc. However, the tool includes the unused skills when calculating turns.  Only way I have gotten it to work is to put delay at 100.  Not sure if there is a different way. 

On a side note, one suggestion would be to turn off the mobile verification for the discord server.  Email verification is fine, but some of us do not let giving out our personal information and they do not allow Google Voice numbers.

**Comment by Dwza** [comment]
just pump up delay...
